desktop/src/main/java/bisq/desktop/app/BisqApp.java

```diff
@@ -231,25 +229,15 @@ private Scene createAndConfigScene(MainView mainView, Injector injector) {
 
         addSceneKeyEventHandler(scene, injector);
 
-        loadSceneStyles(scene, injector);
         injector.getInstance(Preferences.class).getCssThemeProperty().addListener((ov) -> {
-            loadSceneStyles(scene, injector);
+            int cssTheme = injector.getInstance(Preferences.class).getCssTheme();
+            CssTheme.loadSceneStyles(scene, cssTheme);
         });
+        int cssTheme = injector.getInstance(Preferences.class).getCssTheme();
```

Comment:
Can you make a variable for injector.getInstance(Preferences.class) and define int cssTheme above so it can be reused?

Reply:
fixed